sonar-home/src/main/java/org/sonar/home/cache/FileHashes.java

```diff
@@ -49,16 +47,12 @@ public String of(File file) {
    * Computes the hash of given stream. The stream is closed by this method.
    */
   public String of(InputStream input) {
-    try {
+    try(InputStream is = input) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "is" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)